pyproject.toml

```diff
@@ -45,6 +44,23 @@ ruff = ">=0.0.254,<0.3.1"
 coverage = "^7.2.1"
 pytest-cov = "^4.0.0"
 pytest-asyncio = ">=0.20.3,<0.22.0"
+pygobject-stubs = "^2.10.0"
+
+[tool.poetry.group.docs]
+optional = true
+
+[tool.poetry.group.docs.dependencies]
+furo = "*"
+myst-parser = "*"
+myst-nb = "*"
+sphinx = "*"
+sphinx-autobuild = "*"
+sphinx-autodoc-typehints = "*"
+sphinx-copybutton = "*"
+sphinxcontrib-mermaid = "*"
+sphinxcontrib-napoleon = "*"
+sphinx-design  = "*"
+sphinx-inline-tabs = "*"
```

Comment:
Don't use *, use some version boundaries. Generally, when adding dependencies use `poetry add <package>` or `poetry add --group docs <package>` it'll resolve the package and add the latest version tag.

Reply:
Yeah, this needs to be updated thanks for pointing it out. Likely missed as I had all the intention to get GitHub pages up and running with some nice docs.